frappe/utils/data.py

```diff
@@ -374,8 +374,8 @@ def fmt_money(amount, precision=None, currency=None):
 
 	if decimal_str:
 		decimals_after = str(round(amount % 1, precision))
-		parts = decimals_after.split(decimal_str)
-		parts = parts[1] if len(parts) > 1 else ''
+		parts = decimals_after.split('.')
```

Comment:
Splitting at '.' because the separator is always '.' after str conversion in previous step irrespective of number format.